setup.py

```diff
@@ -20,6 +20,7 @@
     package_data={'abz': ['default.conf']},
     scripts=['abzsubmit'],
     data_files=[('bin', ['streaming_extractor_music'])],
+    python_requires='>=2.6,<4',
```

Comment:
like pymbngs, I'm not sure why there's an explicit `<4`. Why do we need to limit it? Surely >=2.6 would be enough here?

I'd be happy to revisit these version numbers and do something like 2.7 and 3.4+. I'll have a look for a table of what versions of python come with common LTS OS releases.

We only supported 2.6 initially because there was one potential contributor who had an old version of osx and didn't know how to use homebrew to install a more recent version... 

Note that this package isn't actually in pypi, so I guess this line does nothing at the moment. However, we should consider adding it eventually, and this is a good config line to have!

Reply:
I already responded about `<4` on the mbngs PR 4 days ago: https://github.com/alastair/python-musicbrainzngs/pull/246#issuecomment-471488060

Basically, by not including `<4`, you're saying that this code will be compatible with Python 4 (whenever that comes around), which is most not likely not going to be a true statement.

Reply:
Answers on https://stackoverflow.com/questions/55186228/should-current-python-projects-list-4-in-python-requires-setup-py-or-not/55188362#55188362 have so far suggested to *not* include the `<4` bit, so I've pushed an amended commit without this now. I will ask in https://github.com/pypa/packaging-problems as well, since the official Python documentation *does* use/include it. I can always make a follow-up PR including it if it turns out to be wanted. :)